dashboard/sql_test.sql

```diff
@@ -110,6 +63,10 @@ GROUP BY author
 ORDER BY stories_created DESC
 LIMIT 5;
 
+SELECT author, story_url FROM stories
+WHERE author LIKE 'PaulHoule'
```

Comment:
Is there a reason we are mentioning a specific author name? Assuming this is in-progress/a draft, because otherwise it might not apply to when records change over time!